src/sentry/models/release.py

```diff
@@ -152,6 +165,10 @@ class Meta:
         app_label = "sentry"
         db_table = "sentry_release"
         unique_together = (("organization", "version"),)
+        # TODO(django2.2): Note that we create this index with each column ordered
+        # descending. Django 2.2 allows us to specify functional indexes, which should
+        # allow us to specify this on the model.
+        index_together = (("organization", "major", "minor", "patch", "revision"),)
```

Comment:
Does this model definition create the correct indexes for tests? Or will you need a shim like what we do for the partial index in BillingHistory in getsentry?

Reply:
Fortunately the indexes it creates by default will just be ascending vs descending and should work fine for tests (tbh, the index is probably not even used in tests since postgres will tend to seq scan small tables). I think this usually just matters for unique indexes.